en/docs/develop/setting-up-user-portal-in-a-dev-environment.md

```diff
@@ -0,0 +1,101 @@
+# Setting-Up User Portal in a Development Environment
+
+In [WSO2 Identity Server 5.10.0](https://wso2.com/identity-and-access-management/) the user portal is available under
+ the URL `http(s)://<SERVER_HOST>/t/<TENANT_DOMAIN>/user-portal`. If you are planning on setting up the repo in a
+ development environment follow the steps given below.
+
+!!! note "Before you begin"
+
+    1. Install [node](https://nodejs.org/en/download/) if you haven’t already(npm is already bundled with node).
+    2. Install [maven](https://maven.apache.org/download.cgi) (Needed to run mvn commands).
+    3. A running Identity Server v 5.10. (If you want to build from source follow the instructions listed [here]
+    (https://github.com/wso2/product-is)).
```

Comment:
A running instance of WSO2 Identity Server (to build from source, follow the instructions given [here]()

Reply:
can we just give a link to the IS binary.zip instead?